com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineSerializedCamera.cs

```diff
@@ -33,14 +35,33 @@ internal class UniversalRenderPipelineSerializedCamera : ISerializedCamera
         public SerializedProperty allowXRRendering { get; }
 #endif
 
+        public (Camera camera, UniversalRenderPipelineSerializedCamera serializedCamera) this[int index]
```

Comment:
TBH I am not really fan of anonymous type. As it simulates somewhat encapsulation, but does not give advantage of tieing data under ceceptual name. Is there a reason why we do not use struct here or saperate methods for accessing?

Reply:
There is no deep reason why I decided to go with the anonymous type here, I thought that creating a class for just two members was little overkill. Let me know if you disagree.

Reply:
I dont know enough here. but maybe we should compare with how we do things somewhere else in our codebase?
Martin may be able to help?

Reply:
@martin-unity3d  do you have some insight over this?

Reply:
no real insight but I think it looks probably more confusing then it is. Which means maybe simplify it like @lukaschod suggested.

Reply:
I've seen this on our codebase, and HDRP is using it a lot and URP is not using at all xD. Maybe we can discuss it usages and see if we should move to proper structs